src/excmds.ts

```diff
@@ -3218,6 +3218,25 @@ export async function clipboard(excmd: "open" | "yank" | "yankshort" | "yankcano
     return done.then(() => undefined)
 }
 
+/** Copy an image to the clipboard.
+
+    @param url
+        Absolute URL to the image to be copied. You can obtain an absolute URL from a relative one using [tri.urlutils.getAbsoluteURL](_src_lib_url_util_.html#getabsoluteurl).
+*/
+//#background
+export async function yankimage(url: string) {
+    const absoluteUrl = UrlUtil.getAbsoluteURL(url, document.baseURI)
+    const image = await window.fetch(absoluteUrl)
+    const blob = await image.blob()
+    // Blob.type returns a MIME type like image/jpeg, but the Clipboard API expects only the second half
+    const imageType = blob.type.split("/")[1]
+    if (!["jpeg", "png"].includes(imageType)){
+        throw new Error("Cannot copy image to clipboard: formats other than JPEG and PNG are not supported")
+    } else {
+        browser.clipboard.setImageData(await blob.arrayBuffer(), imageType as browser.clipboard._SetImageDataImageType)
```

Comment:
What happens if an unsupported image type is given to the this? Presumably it throws a similar error. If that's the case I'd rather we didn't duplicate the error handling as that way if they ever add more formats we'll support it for free.

Reply:
The error thrown is:
> controller in excmd:  Error: Type error for parameter imageType (Invalid enumeration value "gif") for clipboard.setImageData. logging.ts:42:32
log logging.ts:42
acceptExCmd controller.ts:31
It's rather technical, but I expect your average Tridactyl user should be able to cope with that.

Reply:
I'd just wrap it in a try-catch, if imageType is mentioned in the error, throw a new error saying that it's an unsupported filetype and give the mimetype.